src/Illuminate/Database/Query/Builder.php

```diff
@@ -192,6 +193,13 @@ class Builder
      */
     public $useWritePdo = false;
 
+    /**
+     * Common table expressions applied to this query.
+     *
+     * @var CommonTableExpression[]
```

Comment:
FQCN please

Reply:
Ops, sorry. Hmm, anyone an idea how to tell phpstorm not to do this? Doing it by hand is rather tedious…

Reply:
@GrahamCampbell thanks, integrated all your feedback!